tests/scanner_tests.py

```diff
@@ -29,7 +29,7 @@ def test_simple(self):
         self.assertEqual(len(tokens), 3)
         self.assertEqual(tokens[0].name, 'LPAREN')
         self.assertEqual(tokens[0].value, '(')
-        self.assertEqual(tokens[1].name, 'GENERIC_SYMBOL')
+        self.assertEqual(tokens[1].name, 'IDENTIFIER')
```

Comment:
Oh I see where the switch to identifier comes from. This might be a bit misleading. When I think of identifiers I think of variable names.

Reply:
Yeah. The better word might be `'SYMBOL'`, which covers functions and variables. Do you want me to change it, or leave it as is?

Reply:
Hmmm, symbol also has variable connotations in my head. For example that's how you define variables in [pysmt](https://github.com/pysmt/pysmt).

Reply:
Do you want me to change it to something else, or leave it as is?

Reply:
I think leaving it as is is fine